app/components/Views/confirmations/components/SignatureRequest/index.js

```diff
@@ -313,7 +316,7 @@ class SignatureRequest extends PureComponent {
       ),
       external_link_clicked: 'security_alert_support_link',
     };
-    AnalyticsV2.trackEvent(
+    this.props.metrics.trackEvent(
```

Comment:
Destructure metrics on L309
```suggestion
    metrics.trackEvent(
```